Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM ruby:2.3.1
```

Comment:
I knew it was a debian or debian based, but just checked: It's Debian GNU/Linux 8(jessie).